src/main/java/org/terasology/simpleFarming/systems/BushAuthoritySystem.java

```diff
@@ -115,6 +119,11 @@ public void onPlantDestroyed(DoDestroyPlant event, EntityRef entity, BushDefinit
         } else {
             onBudDestroyed(bushComponent, event.isParentDead);
         }
+        EntityRef selected = localPlayer.getCharacterEntity().getComponent(CharacterHeldItemComponent.class).selectedItem;
+        boolean b = selected.getParentPrefab().getName().equalsIgnoreCase("SimpleFarming:gloves");
+        if(b){
```

Comment:
I think this condition should either be inside the if statement (`if(selected.getParentPrefab().`...`)`) (my preference), or it should have a more descriptive name like `glovesUsed`.

Reply:
Well, personally I prefer keeping it short but that'd be a good name indeed.

Reply:
Whilst keeping it short is good, there is no difference between using `b` and `theBooleanThatIndicatesIfTheGlovesWereUsed` to the compiler. Which means that it's better to use something that makes it very clear what the boolean does.

Reply:
By the way try to run Checkstyle on affected files like this, it'll probably both highlight the naming issue as well as some whitespace stuff that helps the code style stay consistent :-)

(There is a Checkstyle plugin for IntelliJ if you haven't set it up yet)